velox/functions/prestosql/window/tests/WindowTestBase.h

```diff
@@ -48,6 +48,28 @@ static std::vector<std::string> kSortOrderBasedOverClauses = {
     "order by c1 desc nulls first, c0 asc nulls first",
 };
 
+/// Common set of window function over clauses for window frame tests to ensure
+/// total ordering for deterministic results in ROW mode.
```

Comment:
Did you mean RANGE mode instead ?

Reply:
Apologies for the confusion but I meant ROW mode, I added the comment so the rationale behind adding another set of overClauses for frame tests is clear.